manifests/tftp.pp

```diff
@@ -10,8 +10,7 @@
     recurse => true;
   }
 
-  foreman_proxy::tftp::sync_file{$foreman_proxy::tftp_syslinux_files:
-    source_path => $foreman_proxy::tftp_syslinux_root,
```

Comment:
tftp_syslinux_root is still defined in the foreman_proxy class parameters, but now doesn't do anything.  I think that's an incompatible change, so we either just accept that and bump the major version, or handle it being defined still.